telegram/src/main/kotlin/com/github/kotlintelegrambot/Bot.kt

```diff
@@ -502,6 +502,7 @@ class Bot private constructor(
         width: Int? = null,
         height: Int? = null,
         caption: String? = null,
+        parseMode: ParseMode? = null,
```

Comment:
This method is deprecated, we shouldn't update it anymore

Reply:
Removed `parseMode` from both deprecated methods